test/index.html

```diff
@@ -4,6 +4,7 @@
 	<meta charset="UTF-8">
 	<title>QUnit Main Test Suite</title>
 	<link rel="stylesheet" href="../dist/qunit.css">
+	<script src="es2017/async-function-tests.js"></script>
```

Comment:
This must be loaded before `../dist/qunit.js`. If it is not, the parse failure on non-async supporting platforms creates a `global failure` test (due to `window.onerror` listener).